src/OpenTK/Math/Vector2.cs

```diff
@@ -623,7 +623,7 @@ public static Vector2 Divide(Vector2 vector, float scale)
         /// <param name="result">Result of the operation.</param>
         public static void Divide(ref Vector2 vector, float scale, out Vector2 result)
         {
-            Multiply(ref vector, 1 / scale, out result);
+            result = vector / scale;
```

Comment:
This is slow - it's passing the values of the vector over to the divide operation.
It's going to perform a full struct copy, and then return.

Fastest would be to perform the raw division per-component directly in the body of the method instead of calling `Multiply`.